### PR TITLE
Tidy stuff

### PR DESCRIPTION
**Pull request by MetRonnie**
This is a small change with no associated Issue.

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Does not need tests
- [x] No change log entry required (invisible to users).
- [x] No documentation update required.
- [x] No dependency changes.

### CODE REVIEW

File: cylc/flow/config.py [hunk 1]
```diff
@@ -666,6 +652,25 @@ def __init__(
...
+        for simplest "R1 = foo" type graphs.
+        """
+        graphdict = self.cfg['scheduling']['graph']
+        if not any(graphdict.values()):
+            raise SuiteConfigError('No suite dependency graph defined.')
+
+        if (
+            'cycling mode' not in self.cfg['scheduling'] and
+            self.cfg['scheduling'].get('initial cycle point', '1') == '1' and
+            all(item in ['graph', '1', 'R1'] for item in graphdict)
```
Comment (MetRonnie) [inline cylc/flow/config.py:666]: This doesn't pick up on equivalent recurrences to `R1`, e.g. `R1/1/1`, but that can come later

File: tests/functional/lib/python/diffr.py [hunk 2]
```diff
@@ -40,10 +40,6 @@
 import json
 import sys
 
-import colorama
-
-colorama.init(strip=True)
-
```
Comment (MetRonnie) [inline tests/functional/lib/python/diffr.py:46]: Unused?

File: tests/functional/lib/python/diffr.py [hunk 3]
```diff
@@ -159,11 +155,8 @@ def tostr(self, indent=0):
...
 
         if indent == 0:
-            # ret += colorama.Fore.GREEN + '+++ %s\n' % self.this_name
-            # ret += colorama.Fore.RED + '--- %s\n' % self.that_name
-            # ret += colorama.Fore.RESET + '============\n'
-            ret += '+++ %s\n' % self.this_name
-            ret += '--- %s\n' % self.that_name
+            ret += '--- %s\n' % self.this_name
+            ret += '+++ %s\n' % self.that_name
             ret += '============\n'
```
Comment (MetRonnie) [inline tests/functional/lib/python/diffr.py:160]: I think these were the wrong way round for the usual diff format

File: tests/unit/test_config.py [hunk 4]
```diff
@@ -302,21 +302,6 @@ def test_queue_config_not_used_not_defined(caplog, tmp_path):
...
-        'scheduling': {
-            'cycling mode': None,
-            'initial cycle point': None
-        }
-    }
-    with pytest.raises(SuiteConfigError) as exc:
-        SuiteConfig.process_initial_cycle_point(mocked_config)
-    assert "This suite requires an initial cycle point" in str(exc.value)
-
-
```
Comment (MetRonnie) [inline tests/unit/test_config.py:319]: This is covered in the test below

File: tests/functional/lib/python/diffr.py (hunk 2 above)
Comment (oliver-sanders) [reply tests/functional/lib/python/diffr.py:46]: I think colour was meant for interactive use but for some reason those lines were commented out. I don't think anyone uses this interactively so whatever.

**Review by oliver-sanders — APPROVED** [review]

**Pull request merged**